label_studio/projects/functions/next_task.py

```diff
@@ -1,27 +1,30 @@
 import logging
 from collections import Counter
+from typing import List, Tuple, Union
 
 from core.feature_flags import flag_set
 from core.utils.common import conditional_atomic, db_is_not_sqlite, load_func
 from django.conf import settings
-from django.db.models import BooleanField, Case, Count, Exists, F, Max, OuterRef, Q, Value, When
+from django.db.models import BooleanField, Case, Count, Exists, F, Max, OuterRef, Q, QuerySet, Value, When
 from django.db.models.fields import DecimalField
 from projects.functions.stream_history import add_stream_history
+from projects.models import Project
 from tasks.models import Annotation, Task
+from users.models import User
 
 logger = logging.getLogger(__name__)
 
 get_tasks_agreement_queryset = load_func(settings.GET_TASKS_AGREEMENT_QUERYSET)
 
 
-def get_next_task_logging_level(user):
+def get_next_task_logging_level(user: User):
```

Comment:
while we're making related changes, the return type of this is apparently `int` (I checked `type(logging.DEBUG)`).